config/routes.rb

```diff
@@ -34,4 +34,5 @@
   end
 
   delete 'sessions/destroy', to: 'sessions#destroy', as: :destroy_session
```

Comment:
I like the change to `post :logout` - could we then remove this mapping (and adjust any places it's used? Probably just the logout button in the main navbar)